src/server/common/model/section/section-model.js

```diff
@@ -3,18 +3,34 @@ import { validateSection } from './validation.js'
 
 /**
  * @import {AnswerModel} from '../answer/answer-model.js'
+ * @import { QuestionPage } from '../page/question-page-model.js'
  */
 
 /**
  * @typedef {{[key:string]: AnswerModel}} SectionPayload
  */
 
 export class SectionModel {
+  /** @type {string}  */
+  title
+
+  /** @type {QuestionPage}  */
+  initialPage
+
+  /** @type {string}  */
+  summaryPageLink
+
+  /** @type {boolean}  */
+  isEnabled = true
```

Comment:
In the current pattern, we're using these fields for configuration that doesn't change amongst object instances - e.g. all instances of this class will have this initialPage & summaryPageLink. 

In this case I don't think that's the case, since we're dynamically enabling this section in some cases.

Reply:
as above, more than happy for this to be a getter, that can dynamically fetch if its enabled or not